include/swift/SIL/SILLinkage.h

```diff
@@ -270,7 +270,7 @@ inline bool isPossiblyUsedExternally(SILLinkage linkage, bool wholeModule) {
 
 SILLinkage getDeclSILLinkage(const ValueDecl *decl);
 
-inline bool hasPublicVisibility(SILLinkage linkage) {
+inline bool hasPublicVisibility(SILLinkage linkage, bool optInPackage) {
```

Comment:
Adding this parameter to `hasPublicVisibility` is highly problematic, because it depends on the context how SILLinkage::Package should be treated when package CMO is enabled.

Reply:
Created a new func hasPublicOrPackageVisibility instead.